### DISCUSSION TITLE
Discord bot auto react messages

### DISCUSSION THREAD

**Discussion by Tesart97** [community/community#47156] [Programming Help]
### Select Topic Area

Question

### Body

Hi all, I'm trying to make a bot that will respond to new messages, it works, but does not see some emoji (don't know what's involved). So, how can I complement the bot to make it respond automatically even to emojis it doesn't know about?

That is, for example, he reacts to emoji :tada: but does not respond to emoji :swords:

```
import requests
from colored import fg
from time import sleep
from os import system


RefreshTime=20

headers = {
    'authorization': "TOKEN HERE"
}



#Main function

links = []

with open('Links.txt') as f:
    for line in f:
        links.append(line.replace("\n", ""))

def Server(links):
    for link in links:
        url = "https://discord.com/api/v9/channels/"+link.split("/")[-1]+"/messages?limit=10"
        response = requests.request("GET", url, headers=headers).json()

        for message in response:
            if "reactions" in message:
                if(message["reactions"][0]["me"] == False):

                    #Need to sleep here cause Discord limits requests
                    sleep(0.3)
                    r = requests.put("https://discord.com/api/v9/channels/"+link.split("/")[-1]+"/messages/"+message["id"]+"/reactions/"+str(message["reactions"][0]["emoji"]['name']+"/%40me"), headers=headers)

                        

while True:
    cls()
    Server(links)
    sleep(int(RefreshTime))
```

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Tesart97 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by aashishvinu** [comment]
@Tesart97 is :swords: a custom added emoji?
